google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITE2ETracingTest.java

```diff
@@ -241,7 +241,7 @@ private boolean dfsContainsCallStack(long spanId, List<String> expectedCallStack
 
   private static final int NUM_SPAN_ID_BYTES = 16;
 
-  private static final int GET_TRACE_RETRY_COUNT = 15;
+  private static final int GET_TRACE_RETRY_COUNT = 60;
```

Comment:
Was 15 not enough?

1 qps of retries. Max allowed is 5 qps. https://cloud.google.com/trace/docs/quotas#:~:text=Trace%20API%20usage%20limits,-API%20method&text=You%20have%20a%20total%20of,API%20calls%20per%2060%20seconds.

Reply:
no it wasn't enough. our backoff is 1 second. so our max qps should be 1.